spec/features/portfolios/user_can_create_portfolio_spec.rb

```diff
@@ -47,7 +47,7 @@
       fill_in 'linkedin', with: "www.test.com"
 
       click_on 'Save as Draft'
-      sleep(1)
+      sleep(5)
```

Comment:
@mollybrown Tried many different things trying to get Capybara to recognize components on the page. I really hope there is a better solution, but so far any better engineered direction I tried I have hit a roadblock.